service/src/test/java/com/ge/predix/acs/monitoring/CacheHealthIndicatorTest.java

```diff
@@ -28,75 +28,100 @@ public void testHealth(final CacheHealthIndicator cacheHealthIndicator, final St
     @DataProvider
     public Object[][] statuses() throws Exception {
         return new Object[][] {
-                new Object[] { mockCacheWithUp(true, true, DecisionCacheHealthIndicator.class), Status.UP,
-                        AcsMonitoringUtilities.HealthCode.AVAILABLE },
-                { mockCacheWithUp(true, true, ResourceCacheHealthIndicator.class), Status.UP,
-                        AcsMonitoringUtilities.HealthCode.AVAILABLE },
-                { mockCacheWithUp(true, true, SubjectCacheHealthIndicator.class), Status.UP,
-                        AcsMonitoringUtilities.HealthCode.AVAILABLE },
-                { mockCacheWithUp(false, true, DecisionCacheHealthIndicator.class), Status.UNKNOWN,
+                new Object[] { mockCacheWithUp(true, true, DecisionCacheHealthIndicator.class,
+                        MockCacheHealthCheckUtil.DECISION), Status.UP, AcsMonitoringUtilities.HealthCode.AVAILABLE },
```

Comment:
Passing cachingEnabled as a parameter to the base cache health indicator class might reduce the number of use cases here.  We only need to test a base class.